### PR TITLE
dmctl: update help message of command

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What problem does this PR solve? 
[docs#261](https://github.com/pingcap/docs-dm/pull/261)

### What is changed and how it works?
- update help message of command

### CODE REVIEW

File: dm/ctl/master/operate_leader.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 func NewOperateLeaderCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "operate-leader <operate-type>",
-		Short: "operate-type can be 'evict' or 'cancel-evict', used to evict leader or cancel evict",
+		Short: "evict/cancel-evict the leader.",
```
Comment (lance6716) [inline dm/ctl/master/operate_leader.go:31]: how about `evict`/`cancel-evict` to indicate it's a part of command (and ditto for below commands)

Comment (GMHDBJD) [reply dm/ctl/master/operate_leader.go]: address in 1caa648

**Comment by lance6716** [comment]
LGTM

File: dm/ctl/master/start_task.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ import (
 func NewStartTaskCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "start-task [-s source ...] [--remove-meta] <config-file>",
-		Short: "start a task as defined in the config file",
+		Short: "Starts a task as defined in the configuration file.",
```
Comment (csuzhangxc) [inline dm/ctl/master/start_task.go:32]: please help me remove `[-s source ...]` in the above line. thanks

File: dm/ctl/master/unlock_ddl_lock.go [hunk 3]
```diff
@@ -29,7 +29,7 @@ import (
 func NewUnlockDDLLockCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "unlock-ddl-lock [-s source ...] <lock-ID>",
-		Short: "forcefully unlock DDL lock",
+		Short: "Unlocks DDL lock forcefully.",
```
Comment (csuzhangxc) [inline dm/ctl/master/unlock_ddl_lock.go:32]: please help me remove `[-s source ...]` in the above line. thanks

**Pull request merged**